app/scripts/modules/core/src/presentation/modal/ModalHeader.module.css

```diff
@@ -7,7 +7,7 @@
 @value closeButtonHoverTransition: all 150ms ease-out;
 
 .header {
-  flex: 1 0 auto;
+  flex: none;
```

Comment:
what's the context on this one?

Reply:
Sorry! I just dropped this PR and didn't even look at it to remember to provide context.

I don't think the header needs to `flex-grow` (but maybe there is a scenario where it should). Right now, if there is not much content, you end up with something like this:
<img width="1402" alt="Screen Shot 2020-01-15 at 9 01 23 PM" src="https://user-images.githubusercontent.com/73450/72494989-6aa7c700-37da-11ea-9ca0-89fcc0d98e38.png">

Reply:
oh weird, I thought that expando business was taken care of but I guess I just didn't deal with it. I _think_ I let it grow because of long modal titles wrapping to multi-line at mobile sizes. If the text wrapping behavior is still there at small widths then it's probably ok.